udata_piwik/templates/piwik.html

```diff
@@ -14,6 +17,25 @@
     var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript';
     g.defer=true; g.async=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
   })();
+
+  {% if config.PIWIK_GOALS %}
+    require(['pubsub'], function(pubsub) {
+      var resourceDownload = {{config.PIWIK_GOALS.RESOURCE_DOWNLOAD}};
+      if (resourceDownload) {
+        pubsub.subscribe('resource-download', function () {
+          console.debug('Tracking goal resource download %s', resourceDownload);
```

Comment:
Using the logger will ensure IE compatibility and application-wide consistent behavior